renovate.json

```diff
@@ -3,15 +3,15 @@
     "extends": [
       "config:base"
     ],
-    "baseBranches": ["main", "release-2.12", "release-2.11", "release-2.10"],
+    "baseBranches": ["main", "release-2.13", "release-2.12", "release-2.11"],
     "postUpdateOptions": [
       "gomodTidy",
       "gomodUpdateImportPaths"
     ],
     "schedule": ["before 9am on Monday"],
     "packageRules": [
       {
-        "matchBaseBranches": ["release-2.12","release-2.11"],
```

Comment:
here also, so we do security update only for release-2.13 and release-2.12 from the moment 2.13 is released